theme/less/_content.less

```diff
@@ -1,32 +1,62 @@
-main article img
+article
 {
-  max-width: 100%;
+  font-size: 1rem;
+  color: #444444;
+  font-weight: 400;
+  line-height: 1.5;
 }
 
-article
+article img
 {
-  color: #444444;
+  max-width: 100%;
 }
 
-article p, article ul, article ol
+article p:first-of-type img
 {
-  font-size: 14px;
-  margin: 0 0 20px 0;
-  line-height: 1.5;
+  margin-top: 1em;
 }
 
 article h1
 {
-  font-size: 35px;
-  font-weight: normal;
+  font-size: 2.19em;
   line-height: 140%;
-  margin-bottom: 14px;
 }
 
 article h2
 {
-  font-size: 28px;
-  font-weight: normal;
+  font-size: 1.75em;
   line-height: 130%;
-  margin-bottom: 10px;
+}
+
+article h3
+{
+  font-size: 1.4em;
+  line-height: 120%;
+}
+
+article h4
+{
+  font-size: 1.12em;
+  line-height: 110%;
+}
+
+article a:hover, article a:visited:hover
+{
```

Comment:
StyleLint test is not passing because of this part. If we putting this part we're getting this error:
`54:1  ×  Expected selector "article a:focus" to come before selector "article a:visited:hover"   no-descending-specificity`
And getting similar error when I put this below it:
`53:1  ×  Expected selector "article a:hover" to come before selector "article a:visited:focus"   no-descending-specificity`

Reply:
Guess we can overwrite the `no-descending-specificity` rule in stylelint.

Reply:
There is similar issue reported here -> https://github.com/stylelint/stylelint/issues/3052 and here -> https://github.com/stylelint/stylelint/issues/2489. They are explaining that this isn't a bug, but I still not sure how we can resolve this issue.